api/Element.json

```diff
@@ -6380,12 +6420,11 @@
                 "version_added": "79"
               },
               {
-                "version_added": "79",
+                "version_added": "12",
```

Comment:
This entry was just a bit messed up, these edits make it consistent with the other changes in this PR.